selftests/functional/serial/test_requirements.py

```diff
@@ -112,12 +113,19 @@ class BasicTest(Test):
         " have it available."
     )
 
+    def setUp(self):
```

Comment:
This is an `avocado.Test` instance, so you can use the readily available `self.workdir` instead.

Reply:
great idea, I will do it, thanks